sdk/vision/Azure.AI.Vision.ImageAnalysis/samples/Sample07_Read.md

```diff
@@ -0,0 +1,68 @@
+# Read An Image
+
+This sample demonstrates how to read text from an image. To get started you'll need a URL for a Computer Vision endpoint. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md) for links and instructions.
+
+## Examples
+
+The following sections provide code snippets using the `client` created above, covering using ImageAnalysis to read text from an image:
+
+### Authenticate the client
+
+In order to interact with Azure Image Analysis, you'll need to create an instance of the [ImageAnalysisClient][imageanalysis_client_class]
+class. To configure a client for use with Azure Image Analysis, provide a valid endpoint URI to an Azure Computer Vision resource
+along with a corresponding key credential authorized to use the Azure Computer Vison resource.
+
+```C# Snippet:ImageAnalysisAuth
+string endpoint = Environment.GetEnvironmentVariable("VISION_ENDPOINT");
+string key = Environment.GetEnvironmentVariable("VISION_KEY");
+
+// Create an Image Analysis client.
+ImageAnalysisClient client = new ImageAnalysisClient(new Uri(endpoint), new AzureKeyCredential(key));
+```
+
+Here we are using enviornment variables to hold the endpoint and key for the Computer Vision Resource.
```

Comment:
enviornment

I use this VS Code tool to spell check source code: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker